### DISCUSSION TITLE
Doubt regarding grain growth

### DISCUSSION THREAD

**Discussion by Rio21092000** [neperfepx/neper#838] [General]
My goal is to show grain growth of a microstructure having 41 grains ,which I already have. I have generated its seed points and weights and kept in files c41.txt and w41.txt respectively. for this i'm using the following code but it is showing error:
neper -T -n 41 -morpho "diameq:lognormal(2,0.70),1-sphericity:lognormal(0.145,0.03)" -domain "cube(10,10,10)" -morphooptiini 'coo:file(c41.txt),weight:file(w41.txt)'                
Error : - Setting seeds... Error  : Failed to read file `file(c41.txt)'.
Can anyone pls elaborate why I am getting error and how can I correct it?

**Comment by rquey** [comment]
To generate a tessellation from the seed properties, use:
```
neper -T -n 41 -domain "cube(10,10,10)" -morphooptiini 'coo:file(c41.txt),weight:file(w41.txt)'
```
From the error message you get, I think that the file isn't properly formatted. It has to have 3 coordinates per line, 41 lines.
